kit/layout/src/Skeleton/Skeleton.tsx

```diff
@@ -213,6 +229,25 @@ export function UISkeleton({
 const styles = StyleSheet.create({
     container: {
         position: 'relative',
+    },
+    skeletonContainer: {
+        /**
+         * It is required so that the content cannot be seen from under the edges of the skeleton
+         */
+        ...Platform.select({
+            web: {
+                top: -1,
+                bottom: -1,
+                left: -1,
+                right: -1,
+            },
+            android: {
+                top: -0.5,
```

Comment:
Actually seems to be fragile in devices with pixel density = 1

Reply:
I agree, this solution has many drawbacks. Deleted. Used `opacity: 0` for children